### DISCUSSION TITLE
Mainsail Release v2.9.1

### DISCUSSION THREAD

**Discussion by meteyou** [mainsail-crew/mainsail#1726] [Announcements]
Ahoi, sailors!

We released v2.9.1 today! (I think this will be the last release for the year 2023 :wink:) In this release, we have fixed some bugs in Mainsail. Here is a short overview of the most important bugfixes:

- Exclude Objects: fix the tooltip position and order of the objects of the map
- Fix language switch for zh-TW (and all future languages with more than 2 chars)
- Fix the Spoolman list with comments and location data
- Fix aspect ratio in Mjpegstreamer-Adaptive mode
- Fix webcam URL with multiple moonraker instances (on one host)
- Fix the tooltip of the temp chart
- Fix a long initial time with a huge print history

The complete changelog for v2.9.1 is available at [https://github.com/mainsail-crew/mainsail/releases/tag/v2.9.1](https://github.com/mainsail-crew/mainsail/releases/tag/v2.9.1).

So, go ahead and hit refresh-button in your update manager and enjoy the new version of Mainsail!

**We wish you all a happy new year and happy printing**

PS: If you want to support the crew's work, consider the following:
- becoming a Patreon: [https://www.patreon.com/meteyou](https://www.patreon.com/meteyou)
- buy us a Ko-Fi: [https://ko-fi.com/mainsail](https://ko-fi.com/mainsail)

**Comment by e-dreyer** [comment]
May it be a great year for Mainsail!

**Comment by QuinnDamerell** [comment]
Amazing! Happy New Year!
